ui/tests/pages/components/form-field.js

```diff
@@ -41,6 +46,9 @@ export default {
     inputChecked: attribute('checked', 'input[type=checkbox]'),
     selectValue: value('select'),
   }),
+  selectRadioInput: async function (value) {
```

Comment:
This seems like it's only used for tests. Is there a different way we could trigger the selection in the test, for example a `click` action on the radio with the desired ID?

Reply:
I was thinking since this is part of form field, we can leverage this for future tests also.